Core/clim-basic/drawing/medium.lisp

```diff
@@ -519,6 +519,10 @@
           :writer (setf %medium-sheet) ))
   (:documentation "The basic class, on which all CLIM mediums are built."))
 
+(defmethod medium-drawable ((medium basic-medium))
+  (alexandria:when-let ((sheet (medium-sheet medium)))
```

Comment:
when-let is already imported in climi package, please remove the package accessor.

Reply:
done.